### PR TITLE
QAE-218 - Test case for Primary menu submenu border radius in the customizer

### PR DESCRIPTION
**Pull request by bsfprajaktap**
Description
This test case is for adding border radius to the submenu

Screenshots
https://share.bsf.io/DOuBdXYr

Types of changes
Test case for adding border radius to submenu

How has this been tested?

1. Create pages.
2. Create menu and sub menu
3. Add border color to the sub menu in customizer settings.

Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by bsfsnehal — APPROVED** [review]

File: tests/e2e/specs/customizer/header-builder/primary-menu-submenu/primary-menu-submenu-border-radius.test.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+		} );
+		await page.hover( '.menu-link' );
+		await expect( {
+			selector: '.sub-menu',
+			property: 'border-radius',
+		} ).cssValueToBe(
+			`${ submenuBorder[ 'header-menu1-submenu-border-radius' ] + 'px' }`,
+		);
+	} );
+} );
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/primary-menu-submenu/primary-menu-submenu-border-radius.test.js:46]: check the  folder structure & filename

File: tests/e2e/utils/create-menu.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import { createURL, createNewPost, publishPost } from '@wordpress/e2e-test-utils';
```
Comment (bsfsnehal) [inline tests/e2e/utils/create-menu.js:1]: check for publish-post util path

File: tests/e2e/specs/customizer/header-builder/primary-menu-submenu/border-radius.test.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import { createURL } from '@wordpress/e2e-test-utils';
+import { createNewMenu } from '../../../../utils/create-menu';
+import { setCustomize } from '../../../../utils/customize';
+describe( 'Add sub menu for primary menu and add border to the sub menu', () => {
+	it( 'sub menu should be added successfully', async () => {
+		await createNewMenu();
+		//center alignment for desktop, tablet and mobile
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/primary-menu-submenu/border-radius.test.js:7]: remove unwanted comment

File: tests/e2e/specs/customizer/header-builder/primary-menu-submenu/border-radius.test.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+				right: 10,
+			},
+		};
+		await setCustomize( submenuBorder );
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.hover( '.menu-link' );
+		await expect( {
+			selector: '.sub-menu',
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/header-builder/primary-menu-submenu/border-radius.test.js:39]: Could you please verify selectors:- https://share.bsf.io/Wnuyl1ml

Comment (bsfsnehal) [reply tests/e2e/specs/customizer/header-builder/primary-menu-submenu/border-radius.test.js:39]: checked.. Looks correct

File: tests/e2e/specs/customizer/header-builder/primary-menu-submenu/border-radius.test.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+		};
+		await setCustomize( headerMenuAlignment );
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '#primary-site-navigation' );
+		await expect( true ).toBe( true );
+	} );
+	it( 'border radius to the submenu should be added correctly', async () => {
+		const submenuBorder = {
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/header-builder/primary-menu-submenu/border-radius.test.js:24]: variable camel case
```suggestion
		const subMenuBorder = {
```

File: tests/e2e/specs/customizer/header-builder/primary-menu-submenu/border-radius.test.js (hunk 4 above)
Comment (GauravKhupse) [reply tests/e2e/specs/customizer/header-builder/primary-menu-submenu/border-radius.test.js:39]: I saw there is the selector for the sub has need to be updated:- https://share.bsf.io/p9u6k7g2

Comment (GauravKhupse) [inline tests/e2e/specs/customizer/header-builder/primary-menu-submenu/border-radius.test.js:39]:
```suggestion
			selector: '.ast-builder-menu-1 .sub-menu',
```

**Review by GauravKhupse — APPROVED** [review]

**Comment by SimranChougule1** [comment]
closing this PR because this option test case already available in another PR

**Pull request closed without merging**